dev-aws/kafka-shared-msk/iam/iam.tf

```diff
@@ -285,3 +285,10 @@ module "iam_credentials_public_events_mapper" {
   consume_topics   = [(kafka_topic.iam_credentials_v1.name)]
   consume_groups   = ["iam.public-events-mapper-iam-credentials_v1"]
 }
+
+module "iam_credentials_public_events_cbc_fraud_detection_consumer" {
```

Comment:
I think that `cbc_fraud_detection_consumer` should do as the module name

Reply:
ah was afraid it would collide with the one in cbc, but if it doesn't I'll change it!